problems/2.functions.js

```diff
@@ -2,36 +2,40 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function greeting(name) {
+  let str = null
+  return str
 }
+console.log(greeting())
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
-
+function greeting(name) {
+  let str = "Hello"
+  return str + name
 }
+console.log(greeting("Avery"))
 
 /** Adds a number to a global variable called globalCount
  * Access globalCount like this: global.globalCount
 * @param  {number} val - The number to add to globalCount
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
+function incrementGlobalCount(val) {}
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val